README.FreeBSD

```diff
@@ -1,8 +1,8 @@
-Target OS: FreeBSD 10.4 (RELEASE) and FreeBSD 11.2 (RELEASE)
+Target OS: FreeBSD 12.2 (RELEASE) and FreeBSD 13.0 (RELEASE)
 
 1. Install necessary packages:
 -------------
-pkg install git cmake gmake readline icu
+pkg install git cmake gmake readline icu libiconv
```

Comment:
Please, leave a note in the commit message regarding libconv problem that we discussed privately. Ideally also link a relevant issue.

Reply:
done, comment updated